BusBoard.ConsoleApp/Program.cs

```diff
@@ -25,22 +25,17 @@ static void Main(string[] args)
 
             var coordinate = postcodeApiHelper.GetPostcodeCoordinates(input);
             var stopPoints = tflApiHelper.GetStopPoints(coordinate);
-            if (stopPoints == null)
-            {
-                Console.WriteLine("No nearby stops found for that postcode!");
-            }
+            var output = new List<string>();
 
-            else
+            if (stopPoints != null)
```

Comment:
I you change this to have stopPoints as an empty list instead of `null` if there are no stops then that will simplify your code and make the logic easier to follow 🙂

Reply:
Are you suggesting to have the null check within the GetStopPoints() method so that it outputs an empty list rather than null?

Reply:
Yes 🙂